GameServerLib/Logic/ObjectManager.cs

```diff
@@ -380,5 +380,29 @@ public bool TeamHasVisionOn(TeamId team, GameObject o)
 
             return false;
         }
+
+        public int CountUnitsAttackingUnit(AttackableUnit target)
+        {
+            int count = 0;
+            try
+            {
+                var enemyUnits = GetObjects();
+                foreach (var obj in enemyUnits)
+                {
+                    if(obj.Value is ObjAiBase aiBase)
+                    {
+                        if(aiBase != null && aiBase.Team != target.Team && aiBase.Team != TeamId.TEAM_NEUTRAL && !aiBase.IsDead && aiBase.TargetUnit != null && aiBase.TargetUnit == target)
```

Comment:
Lines should not be longer than 120 characters. You might want to split this line (and a few others you have changed/added if they are off limits, too) into multiple lines.